frontend/src/components/Team.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {
+  Card, CardImg, CardTitle, CardText,
+  CardSubtitle, CardBody, Row, Col
+} from 'reactstrap';
+import '../styles/Team.css';
+import data from '../jsonData/TeamData.json'
+
+const cardStyle = {
+  backgroundColor: '#E4A58F',
+  borderWidth: 'thick',
+  borderColor: '#333',
+}
+
+function Team() {
+  return (
+    <div id="team">
+      <Row>
+        {data.map((person, index) => (
```

Comment:
So cool that you used the map function here!!!!!

Reply:
Thank you for suggesting it!